src/modules/auth/pages/Register.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect } from 'react';
+import { Input, Button, Row, Col, Layout, Form, message } from 'antd';
+import { Link } from 'react-router-dom';
+import Cookies from 'js-cookie';
+import AuthAPIClient from '../services/AuthAPIClient';
+import Footer from '../../../core/components/Footer';
+import '../style/auth.scss';
+import logo from '../../../assests/images/logo.jpeg';
```

Comment:
```suggestion
import logo from '../../../assets/images/logo.jpeg';
```

Reply:
I will create a task to rename assets folder

Reply:
Do it in this PR. We dont have that much time anymore